lib/banyan_api/storefront_event.ex

```diff
@@ -2,21 +2,23 @@ defmodule BanyanAPI.StorefrontEvent do
   alias BanyanAPI.Client
 
   @spec track(map(), String.t(), map()) :: {:ok, %Neuron.Response{}} | {:error, any()}
-  def track(shop, type, %{shopify_order: shopify_order}) do
+  def track(shop, type, %{shopify_order: shopify_order, recipient_email: recipient_email}) do
```

Comment:
what if we left it out of the func def and pulled it from the map where needed?

Reply:
The should probably be done for the `shopify_order` as well. I assume we'll have some storefront events that don't involve an order.

Reply:
Yeah, I was following the pattern established, but I agree -- we talked about using pattern matching to handle different "types" of storefront events, but if so that would probably happen upstream rather than here.

Should I change `shopify_order` while we're at it? Basically any fields that could be optional in the future.

Reply:
Heh.